src/components/responsive/MobileView.js

```diff
@@ -1,321 +0,0 @@
-                                        </Menu.Item>
-                                    </Link>
-                                */}
-                                <Link to='/authorized-apps' onClick={this.handleDropdown}>
-                                    <Menu.Item>
-                                        <Image src={AuthorizedGreyImage} />
-                                        <Translate id='desktopPopup.authorizedApps' />
-                                    </Menu.Item>
-                                </Link>
-                                <Link to='/full-access-keys' onClick={this.handleDropdown}>
-                                    <Menu.Item>
-                                        <Image src={KeysImage} />
-                                        <Translate id='desktopPopup.fullAccessKeys' />
-                                    </Menu.Item>
-                                </Link>
-                                {/*
-                                    <Link to='/' onClick={this.handleDropdown}>
-                                        <Menu.Item>
-                                            <Image src={LogoutImage} />
-                                            <Translate id='desktopPopup.logout' />
-                                        </Menu.Item>
-                                    </Link>
-                                */}
-                            </Menu.Menu>
-                            <Segment basic className='switch-account'>
-                                <List>
-                                    <List.Item as='h6'><Translate id='desktopPopup.switchAccount' /></List.Item>
-                                    {availableAccounts
-                                        .filter(a => a !== account.accountId)
-                                        .map((account, i) => (
-                                            <List.Item
-                                                as='a'
-                                                key={`mf-${i}`}
-                                                onClick={() => this.handleSelectAccount(account)}
-                                                className='account-title'
-                                            >
-                                                @{account}
-                                            </List.Item>
-                                        ))}
-                                    {availableAccounts.length < 2 && <Translate id='desktopPopup.noAccount' />}
-                                </List>
-                                <Button onClick={this.redirectCreateAccount}>
-                                    <Translate id='button.createNewAccount' />
-                                </Button>
-                            </Segment>
-                        </Menu.Menu>
-                    </Segment>
-                </Segment>
-                <GlobalAlert />
```

Comment:
@Patrick1904 I think you forgot about GlobalAlert, it's not appearing in the Wallet now.

Reply:
@marcinbodnar sorry I forgot to tell you that I left it out. It's being used properly in LoginDetails and a few other components. Seems suboptimal to default errors/successes to an inline 'GlobalAlert' at the top of the page, pushing down the rest of the content. It's quick/easy but UX suffers. A global 'status/flag' to reference makes sense. It makes sense to put a reusable fixed position components in Routing, like a modal, snackbar (I should have done that) since they can be added to any page regardless of the page content.

What is the current case that you're going to use the GlobalAlert for? Maybe there's a better way to display error/success in this case

Reply:
@marcinbodnar thanks for clarifying. I guess it’s the solution requiring the least effort right now. Reg login success, just note that we recently disabled the GlobalAlert since it only appeared for short sec before re-direct making it appear like a bug.   You can add the GlobalAlert component to Routing.js and it'll be available on every page. The nav is fixed so can’t be inside the nav component. You can put it below it and should work fine.

Reply:
@Patrick1904 If that's not a problem, it would be better if you would add it since you were this part of the project and you know better where to put it now.

Reply:
Sure thing here's the commit: https://github.com/nearprotocol/near-wallet/pull/497

Reply:
cool thank you